lacci/lacci.gemspec

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative "../lib/scarpe/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "lacci"
+  spec.version = Scarpe::VERSION
```

Comment:
I like binding this to the scarpe version

Reply:
Yeah, I think that's a good idea for some time yet. Eventually we're going to need to distinguish between "Scarpe, the Webview-based display library" and "Scarpe, the layer of display implementation for Lacci, especially logging and testing code." In the long run they're not the same thing. Wasm is going to push us into splitting it into more pieces long-term. All these divisions were kind of theoretical, right up until they weren't :-)